linter.py

```diff
@@ -84,16 +84,14 @@ def reposition_match(self, line, col, m, vv):
         match = m.match
         if (
             col is None or
-            match.get('fatal', False)  # parse error
```

Comment:
Would this ever need to be handled, or is it something either so rare that it can be ignored, or so common that SublimeLinter/linter.py should be handling it itself?

Reply:
`fatal` errors are actually parse errors. You can produce them easily.

Before this change I marked the whole line, with this change I mark only the one character leading to the parse error. (fatal errors don't have an end, so they actually still match this if branch.)

Reply:
Ah, makes sense. Thanks @kaste